.gitignore

```diff
@@ -1,4 +1,4 @@
-userConfig.h
+userConfig.h*
```

Comment:
Hi Nils,
the pattern is not correct because there is a userConfig.h.SAMPLE which needs to be included.

Reply:
`userConfig.h.SAMPLE` will be unaffected as it is already in the index, only new files will be ignored.

Reply:
you are right, but still I dont want to depend on a "pre-existing" index.
Just adapt the pattern to a working state, please. (https://labs.consol.de/de/development/git/2017/02/22/gitignore.html)